src/LevelSetCutters/DiscreteGeometries.jl

```diff
@@ -65,3 +65,9 @@ function _find_unique_leaves(tree)
   j_to_fun, oid_to_j
 end
 
+function discrete_geometry(
```

Comment:
Can you change the name of this function to `DiscreteGeometry`? 
it is just a constructor of a `DiscreteGeometry` object. This is why we want it in camel case.